docs/root/start/sandboxes/locality_load_balancing.rst

```diff
@@ -0,0 +1,153 @@
+.. _install_sandboxes_locality_load_balancing:
+
+Locality Weighted Load Balancing
+================================
+
+.. sidebar:: Requirements
+
+   .. include:: _include/docker-env-setup-link.rst
+
+   :ref:`curl <start_sandboxes_setup_curl>`
+        Used to make ``HTTP`` requests.
+
+This example creates a setup for demonstrating the :ref:`locality weighted load balancing <arch_overview_load_balancing_locality_weighted_lb>` feature in Envoy proxy. The demo simulates a scenario that a backend service resides in two local zones and one remote zone.
+
+The components used in this demo are as follows:
+
+- A client container: runs Envoy proxy
+- Backend container in the same locality as the client, with priority set to 0, namely ``local-1``.
+- Backend container in the same locality as the client, with priority set to 1, namely ``local-2``.
+- Backend container in the the remote locality, with priority set to 1, namely ``remote-1``.
+- Backend container in the the remote locality, with priority set to 2, namely ``remote-2``.
+
+The client Envoy proxy configures the 4 backend containers in the same Envoy cluster, so that Envoy handles load balancing to those backend servers. From here we can see, we have localities with 3 different priorities:
+
+- priority 0: ``local-1``
+- priority 1: ``local-2`` and ``remote-1``
+- priority 2: ``remote-2``
+
+In Envoy, when the healthiness of a given locality drops below a threshold (71% by default), the next priority locality will start to share the request loads. The demo below will show this behavior.
+
+Step 1: Start all of our containers
+***********************************
+
+In terminal, move to the ``examples/locality_load_balancing`` directory.
+
+To build this sandbox example and start the example services, run the following commands:
+
+.. code:: shell
+
+    # Start demo
+    $ docker-compose up --build -d
+
+The locality configuration is set in the client container via static Envoy configuration file. Please refer to the :download:`clusters configuration <_include/locality-load-balancing/envoy-proxy.yaml>` file.
```

Comment:
perhaps "...refer to the ``clusters`` section of the :download: proxy configuration <..."